tests/python/pants_test/test_base.py

```diff
@@ -516,9 +516,19 @@ def tearDown(self):
     super(TestBase, self).tearDown()
     Subsystem.reset()
 
+  @classproperty
+  def target_class_for_subsystems(cls):
```

Comment:
Would exposing a generic `def subsystems` method (and defaulting it to `Target.subsystems()`) be more general?

Reply:
Sounds like it would!